src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js

```diff
@@ -936,6 +936,165 @@ Vue.component('sortIndicator', {
     }
   }
 });
+// <upload> component
+Vue.component('upload', {
+  template: '\
```

Comment:
@agriffard I might be missing something, but it seems like the `.media.min.js` file didn't get a rebuild here ?